### PR TITLE
Add interaction to immediatly stop a land or maritime transport

### PR DESCRIPTION
**Pull request by Mysteryjuju**
Option is available only when the vehicle is moving.

### CODE REVIEW

File: addons/common/functions/fnc_stopVehicle.sqf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+params ["_entity"];
+
+if (isNull _entity) exitWith {};
+
+private _vehicle = _entity getVariable ["SSS_vehicle",objNull];
+ 
+INTERRUPT(_entity,_vehicle);
+
+// Stop the vehicle and delete all waypoints
+[_vehicle] call CBA_fnc_clearWaypoints;
```
Comment (SceptreOfficial) [inline addons/common/functions/fnc_stopVehicle.sqf:16]: `FUNC(clearWaypoints)` for special stuff that may need to be handled.

**Comment by SceptreOfficial** [comment]
Do you think it should be "Immediate Stop" or just "Stop"?

**Comment by Mysteryjuju** [comment]
As you want, english is not my natural language. What is the best choice for you ? In my own opinion, the two solution are goods.

**Comment by SceptreOfficial** [comment]
Either is fine. I think I would say "stop" for less screen clutter, and I see you updated it anyway. I'll merge for now, thanks.

**Pull request merged**